dcp/CheckpointManager.go

```diff
@@ -359,7 +360,7 @@ func (cm *CheckpointManager) getStatsWithRetry() (map[string]map[string]string,
 							singleServerStats.Error)
 						continue
 					}
-					cm.logger.Debugf("Server %v received stats %v", server, singleServerStats.Stats)
+					cm.logger.Infof("Server %v received stats %v", server, singleServerStats.Stats)
```

Comment:
This could be spammy because the Stats returned is a large block of text.
Given the err check in 376, could we potentially print the content of `result.Servers` if the err in line 376 is non-nil to aid in diagonistic?